CSS/style.css

```diff
@@ -1502,9 +1503,9 @@ textarea {
     margin-bottom: -500px;
   }
 
-   /* ============================Start of Desktop-Details Popup Window======================= */
+  /* ============================Start of Desktop-Details Popup Window======================= */
 
-   .modal-content {
+  .modal-content {
```

Comment:
- [ ]Kindy remove the border around the popup image and also ensure that the width of the image span across the popup container to match your chosen Figma design  😄 . 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181957093-fb0b37cc-8126-417d-92e1-6f73db61d3be.png)  | ![image](https://user-images.githubusercontent.com/63473584/181958772-3ebaaee2-8530-451a-b5a4-3dab06e342b3.png)  |

Reply:
Thank you for the positive review